final_task/rss_reader/article.py

```diff
@@ -0,0 +1,75 @@
+from string_operations import *
+import logging
+
+
+class Article:
+    """single article class"""
+
+    def __init__(self, parsed):
+        """receive parsed article and extracts data from it"""
+        logging.info('      title')
+        self.title = make_string_readable(parsed.title)
+
+        logging.info('      link')
+        self.link = parsed.link
+
+        logging.info('      date')
+        self.published = extract_date(parsed)
+
+        logging.info('      summary')
+        summary_ = extract_topic_info_from_summary(parsed.summary)
+        self.summary = make_string_readable(summary_)
+
+        logging.info('      image links')
+        self.media = parsed.media_content
+
+        logging.info('      image description')
+        description_ = extract_image_info_from_summary(parsed.summary)
+        self.media_description = make_string_readable(description_)
+
+    def print_readable_article(self):
+        """print article to stdout in human-readable format"""
+
+        # separation line
+        print("_" * 79)
+
+        # date
+        print(self.published[:-6], ':\n')
```

Comment:
It is better to use `datetime` for this instead of applying slices to string date, because depending on rss source, they may use different date formats.

Reply:
Fixed